packages/styling/src/styles/theme.ts

```diff
@@ -1,19 +1,12 @@
-import { Customizations, merge, getWindow } from '@uifabric/utilities';
-import { IPalette, ISemanticColors, ITheme, IPartialTheme, IFontStyles, IEffects } from '../interfaces/index';
-import { DefaultFontStyles } from './DefaultFontStyles';
-import { DefaultPalette } from './DefaultPalette';
-import { DefaultSpacing } from './DefaultSpacing';
+import { Customizations, getWindow } from '@uifabric/utilities';
+import { ITheme, IPartialTheme, IFontStyles } from '../interfaces/index';
 import { loadTheme as legacyLoadTheme } from '@microsoft/load-themed-styles';
-import { DefaultEffects } from './DefaultEffects';
 import { IRawStyle } from '@uifabric/merge-styles';
+import { createTheme } from '@fluentui/theme/lib/createTheme';
 
-let _theme: ITheme = createTheme({
-  palette: DefaultPalette,
-  semanticColors: _makeSemanticColors(DefaultPalette, DefaultEffects, undefined, false, false),
-  fonts: DefaultFontStyles,
-  isInverted: false,
-  disableGlobalClassNames: false,
-});
+export { createTheme } from '@fluentui/theme/lib/createTheme';
+
+let _theme: ITheme = createTheme({});
```

Comment:
Confirmed that the previous code is the same as `createTheme({})` [here](https://codesandbox.io/s/angry-http-97rug)